lib/Screens/chat_screen.dart

```diff
@@ -1,12 +1,211 @@
 import 'package:flutter/material.dart';
+import '../Utils/Styles.dart';
+import './conversations.dart';
+
+class ChatScreen extends StatefulWidget {
+  @override
+  _ChatScreenState createState() => _ChatScreenState();
+}
+
+class _ChatScreenState extends State<ChatScreen> {
+  List<ChatModel> chatList;
+
+  @override
+  void initState() {
+    super.initState();
+    chatList = GetChatData();
+  }
+
+  List<ChatModel> GetChatData() {
+    List<ChatModel> list = List<ChatModel>();
+    var content1 = ChatModel();
+    content1.name = "Alice Smith";
+    content1.msg = "Hi Alice How are you";
+    content1.img = "http://tny.im/oz7";
+    content1.isOnline = true;
+    list.add(content1);
+
+    var content2 = ChatModel();
+    content2.name = "Hennah Tran";
+    content2.msg = "Hi can u explain me this topic?";
+    content2.img = "http://tny.im/oz6";
+    list.add(content2);
+
+    var content3 = ChatModel();
+    content3.name = "Hennah Tran";
+    content3.msg = "Hi can u explain me this topic?";
+    content3.img = "http://tny.im/oz7";
+    list.add(content3);
+
+    var content4 = ChatModel();
+    content4.name = "Hennah Tran";
+    content4.msg = "Hi can u explain me this topic?";
+    content4.img = "http://tny.im/oz6";
+    content4.isOnline = true;
+    list.add(content4);
+
+    return list;
+  }
 
-class ChatScreen extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
-    return Scaffold(
-      body: Center(
-        child: Text('Chat Screen'),
+    final listInTab = SingleChildScrollView(
+      child: Column(
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: <Widget>[
+          ListView.builder(
+            scrollDirection: Axis.vertical,
+            itemCount: chatList.length,
+            shrinkWrap: true,
+            physics: NeverScrollableScrollPhysics(),
+            itemBuilder: (context, index) {
+              return ChatsData(chatList[index], index);
+            },
+          ),
+        ],
       ),
     );
+
+    return Scaffold(
+        appBar: PreferredSize(
+          preferredSize: Size.fromHeight(70),
+          child: new Container(
```

Comment:
The container doesn't fit the full width of the device.

Reply:
🤔it takes the whole width in my screen... anyways i''ll give it mediaquery whole width and make the commit soon by today

Reply:
made the changes u asked for :)